{"name": "Cannot update the Week start day with Start_date in the project with wrong values -- @1.2 ", "status": "passed", "steps": [{"name": "Given I set up a \"PUT\" request to \"/projects/{proj_id}\" endpoint", "status": "passed", "start": 1556562465569, "stop": 1556562465570}, {"name": "And I set up the data", "status": "passed", "attachments": [{"name": ".text", "source": "49cbffa1-f260-4f83-9ed4-b12716291a25-attachment.txt", "type": "text/plain"}], "start": 1556562465570, "stop": 1556562465572}, {"name": "When I send the request", "status": "passed", "start": 1556562465572, "stop": 1556562466041}, {"name": "Then I get a \"400\" status code as response", "status": "passed", "start": 1556562466041, "stop": 1556562466043}, {"name": "And I verify the general_problem of error is: \"'start_date' must be a date value represented as iso 8601 date\"", "status": "passed", "start": 1556562466043, "stop": 1556562466044}], "parameters": [{"name": "day", "value": "SUNDAY"}, {"name": "date", "value": "21-04-2019"}, {"name": "message", "value": "'start_date' must be a date value represented as iso 8601 date"}], "start": 1556562465567, "stop": 1556562466044, "uuid": "d3979934-e04f-4b5d-8fc8-84dccc6881a6", "historyId": "27362844c7cc9cdb7a1e144200b13d94", "labels": [{"name": "tag", "value": "negative"}, {"name": "tag", "value": "project"}, {"name": "severity", "value": "normal"}, {"name": "feature", "value": "Functional Test Negative Cases of Update Projects"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython3"}]}